core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/base/DatabaseBase.java

```diff
@@ -11,11 +12,22 @@
     @NotNull
     private DatabaseAvailabilityType availabilityType;
 
+    @Pattern(regexp = "^(9\\.6|10|11|12|13|14)?$", message = "Not a valid database major version")
```

Comment:
Could you pls explain what's the rationale of storing only the major version over storing the full version here?

Reply:
also this regexp pattern is duplicated 3 times, I'm worried if it needs modification only 1 or 2 would be updated. Could you move this to something common?

Reply:
In Azure only the major version can be specified, so we currently do not support sendin gin the minor version